cnchi/installation/install.py

```diff
@@ -774,10 +774,10 @@ def update_pacman_conf():
                     else:
```

Comment:
No it shouldnt be. Maybe there is a typo or something. I cant check it now. But the code I provided specifically writes the testing line after writing the antergos lines.